ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Create OpenStack namespace
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: namespace
+        params: &install_yamls_out
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+
+    - name: Generate olm artifacts
```

Comment:
you do not need to do this as a speerate step
it will be one via the invocatoin of the openstack target

Reply:
I had added those tasks initially while developing this job. I have updated the pull request to remove extra make invocation and keep only openstack and openstack_deploy target.